modules/system/classes/UpdateManager.php

```diff
@@ -549,7 +552,7 @@ public function rollbackPlugin($name)
             return $this;
         }
 
-        if ($this->versionManager->removePlugin($plugin)) {
+        if ($this->versionManager->removePlugin($plugin, $stopOnVersion)) {
             $this->note('<info>Rolled back:</info> ' . $name);
```

Comment:
Should this also state what version the plugin was rolled back to if there was one set?

Reply:
![Screen Shot 2019-06-17 at 20 48 17](https://user-images.githubusercontent.com/3984760/59643985-5666f300-9141-11e9-85e0-2abae62ef08c.png)